_automating-configurations/workflow-steps.md

```diff
@@ -52,6 +52,7 @@ You can include the following additional fields in the `user_inputs` field when
 |`node_timeout`	|Time units	|A user-provided timeout for this step. For example, `20s` for a 20-second timeout.	|
 |`deploy`	|Boolean	|Applicable to the Register Model step type. If set to `true`, also executes the Deploy Model step.	|
 |`tools_order`	|List	|Applicable only to the Register Agent step type. Specifies the ordering of `tools`. For example, specify `["foo_tool", "bar_tool"]` to sequence those tools in that order.	|
+|`delay`	|Time units	|Applicable to the No-op step type. Waits the specified time. For example, `250ms` will wait for 250 milliseconds before continuing the workflow.	|
```

Comment:
Does this change the technical meaning?

```suggestion
| `delay` | Time units | Sets the specified time to wait during a no-op step.  For example, when set to `250ms`, the no-op step waits for 250 milliseconds before continuing the workflow. |
```

Reply:
> Does this change the technical meaning?

The steps in this table are only applicable to some steps.  The two entries above this are specific only for the Register Model Step, for example, so we should address these consistently.  Perhaps a new column in the table for "Steps applicable to" ?

Reply:
I think an extra column in the table might make sense. Let me take a pass at it.